.github/workflows/build-triton-wheel.yml

```diff
@@ -157,8 +156,15 @@ jobs:
       - name: Download Build Artifacts
         uses: actions/download-artifact@v3
         with:
-          name: pytorch-triton-wheel
-          path: ${{ runner.temp }}/artifacts/
+          # Download all available artifacts
+          path: ${{ runner.temp }}/artifacts-all
```

Comment:
Just curious on why you need to download them all to `artifacts-all`.  Is that to make sure that the `artifacts` directory structure remains the same so that the upload script could find all the `*.whl` and `*.tar.bz2` files correctly?

Reply:
That's correct. When you use `download-artifact` without specifying `name` it downloads all artifacts into their own subdirectory, whereas the upload script expects a flat directory structure. 

This step selects the relevant folders and flattens them into the `artifacts` directory to recover the previous directory structure and contents.

On newer versions of the `download-artifact` action this is a lot simpler as you can specify:
```
pattern: pytorch-triton-wheel-*
merge-multiple: true
```

Unfortunately the newer version isn't supported on amazon linux 2 so I did this as a workaround.